### PR TITLE
[release-0.20] changelog for v0.20.8

### PR DESCRIPTION
**Pull request by abhinavmpandey08**
*Description of changes:*
Changelog for v0.20.8

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -269,6 +297,37 @@ description: >
...
+- etcdadm-controller: `v1.0.23` to `v1.0.24`
+- etcdadm-bootstrap-provider:  `v1.0.13` to `v1.0.14`
+- local-path-provisioner: `v0.0.29` to `v0.0.30`
+- runc: `v1.1.14` to `v1.1.15`
+
+### Fixed
+- Skip hardware validation logic for InPlace upgrades. [#8779](https://github.com/aws/eks-anywhere/pull/8779)
+- Status reconciliation of etcdadm cluster in etcdadm-controller when etcd-machines are unhealthy. [#63](https://github.com/aws/etcdadm-controller/pull/63)
+- Skip generating AWS IAM Kubeconfig on cluster upgrade. [#8851](https://github.com/aws/eks-anywhere/pull/8851)
+
```
Comment (sp1999) [inline docs/content/en/docs/whatsnew/changelog.md:330]: Let's add `v0.19.9` and `v0.19.10` too for consistency

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -24,12 +24,40 @@ description: >
...
+  * [CVE-2024-9594: VM images built with Image Builder with some providers use default credentials during builds](https://github.com/kubernetes/kubernetes/issues/128007)
```
Comment (2ez4szliu) [inline docs/content/en/docs/whatsnew/changelog.md:28]:
```suggestion
    * [CVE-2024-9594: VM images built with Image Builder with some providers use default credentials during builds](https://github.com/kubernetes/kubernetes/issues/128007)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -24,12 +24,40 @@ description: >
...
   * [Upgrading an EKS Anywhere cluster]({{< relref "../clustermgmt/cluster-upgrades" >}})
+  * On October 11, 2024, a security issue CVE-2024-9594 was discovered in the Kubernetes Image Builder where default credentials are enabled during the image build process when using the Nutanix, OVA, QEMU or raw providers. The credentials can be used to gain root access. The credentials are disabled at the conclusion of the image build process. Kubernetes clusters are only affected if their nodes use VM images created via the Image Builder project. Clusters using virtual machine images built with [Kubernetes Image Builder](https://github.com/kubernetes-sigs/image-builder) version `v0.1.37` or earlier are affected if built with the Nutanix, OVA, QEMU or raw providers. These images built using previous versions of image-builder will be vulnerable only during the image build process, if an attacker was able to reach the VM where the image build was happening, login using these default credentials and modify the image at the time the image build was occurring. This CVE has been fixed in image-builder versions >= `v0.1.38`, which has been included in EKS Anywhere release `v0.19.11`.
```
Comment (2ez4szliu) [inline docs/content/en/docs/whatsnew/changelog.md:27]:
```suggestion
* On October 11, 2024, a security issue CVE-2024-9594 was discovered in the Kubernetes Image Builder where default credentials are enabled during the image build process when using the Nutanix, OVA, QEMU or raw providers. The credentials can be used to gain root access. The credentials are disabled at the conclusion of the image build process. Kubernetes clusters are only affected if their nodes use VM images created via the Image Builder project. Clusters using virtual machine images built with [Kubernetes Image Builder](https://github.com/kubernetes-sigs/image-builder) version `v0.1.37` or earlier are affected if built with the Nutanix, OVA, QEMU or raw providers. These images built using previous versions of image-builder will be vulnerable only during the image build process, if an attacker was able to reach the VM where the image build was happening, login using these default credentials and modify the image at the time the image build was occurring. This CVE has been fixed in image-builder versions >= `v0.1.38`, which has been included in EKS Anywhere release `v0.19.11`.
```

**Review by 2ez4szliu — COMMENTED** [review]
sorry that previous suggestion was wrong, so I think this line `On October 11, ...` should be on the same indentation as line 21 `On January 31, 2024...`

File: docs/content/en/docs/whatsnew/changelog.md [hunk 4]
```diff
@@ -269,6 +297,83 @@ description: >
...
+- Skip generating AWS IAM Kubeconfig on cluster upgrade. [#8851](https://github.com/aws/eks-anywhere/pull/8851)
+
+
+## [v0.19.10](https://github.com/aws/eks-anywhere/releases/tag/v0.19.10)
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.22.0 |    ✔    |     \*      |    —    |     —      |  —   |
```
Comment (sp1999) [inline docs/content/en/docs/whatsnew/changelog.md:338]:
```suggestion
| Bottlerocket 1.20.5 |    ✔    |     \*      |    —    |     —      |  —   |
```

**Pull request merged**